config/manifests/bases/nfd-operator.clusterserviceversion.yaml

```
@@ -463,8 +463,8 @@ spec:
                 - name: OPERATOR_NAME
                   value: cluster-nfd-operator
                 - name: NODE_FEATURE_DISCOVERY_IMAGE
-                  value: k8s.gcr.io/nfd/node-feature-discovery:master
-                image: k8s.gcr.io/nfd/node-feature-discovery-operator:master
+                  value: k8s.gcr.io/nfd/node-feature-discovery:v0.11.0-minimal
```

Needs to be master, too
```suggestion
                  value: k8s.gcr.io/nfd/node-feature-discovery:master-minimal
```

---

This registry doesn't hold a master tag. Only staging has a master tag.
Will make the change and push